.travis.yml

```diff
@@ -17,7 +17,7 @@ matrix:
     env: BUILDTYPE=Release
 script: "./travis-compile.sh"
 install: "./travis-dependencies.sh"
-cache: apt
+cache: ccache
```

Comment:
Please check my link above: https://docs.travis-ci.com/user/caching/#ccache-cache

We have to add this to make it work on mac. Wasn't sure where to put it or if there some mac specific lines are needed. Feel free to add it! (that's why i cc'ed you :P)